utils/change_engine/ordinary_to_atomic.py

```diff
@@ -1,83 +1,75 @@
+                f'Changing for database {ordinary_database.name} failed in previous launch.\n'
+                 'Do you want to continue changing? (y/n)\n'
+            )
+
+            if answer == 'y':
+                atomic_database.move_tables(ordinary_database.tables)
+                ordinary_database.drop()
+                atomic_database.rename(ordinary_database.name)
+            elif answer == 'n':
+                if not ordinary_database.exists:
+                    ordinary_database.create()
+
+                ordinary_database.move_tables(atomic_database.tables)
+                atomic_database.drop()
             else:
-                self.__continue_change(atomic_name)
-                print('Changing continued after fail in previous launch, atomic table was restored')
-            return
-
-        self.__client.execute(f'CREATE DATABASE {atomic_name} ENGINE=Atomic')
-        return atomic_name
-
-    def __rename_table(self, old_name, new_name):
-        self.__client.execute(f'RENAME TABLE {old_name} TO {new_name}')
+                self.__user_interactor.notify(f'Wrong answer, skipped')
 
-    def __drop_database(self, db_name):
-        self.__client.execute(f'DROP DATABASE {db_name}')
-
-    def __remove_atomic_prefix(self, name):
-        assert atomic_name.startswith(self.__atomic_prefix)
-
-        prefix_len = len(self.__atomic_prefix)
-        return name[prefix_len:]
-
-    def __rename_atomic_db(self, atomic_name):
-        ordinary_name = self.__remove_atomic_prefix(ordinary_name)
-        self.__client.execute(f'RENAME DATABASE {atomic_name} TO {ordinary_name}')
+            return False
+        else:
+            return True
 
     def convert(self, ordinary_name):
-        assert self.__db_exist(ordinary_name), 'ordinary database does not exist'
-        assert self.__db_engine(ordinary_name) == 'Ordinary', 'something went wrong'
+        ordinary_database = Database(ordinary_name, self.__client)
+        atomic_database = Database(self.__get_atomic_name(ordinary_name), self.__client)
+
+        if self.__finished_previous_sessions(ordinary_database, atomic_database):
+            if not ordinary_database.exists:
+                raise ValueError(f'ordinary database \'{ordinary_database.name}\' does not exists')
 
-        tables = self.__get_tables(ordinary_name)
-        atomic_name = self.__create_atomic_db(ordinary_name)
+            if ordinary_database.engine != 'Ordinary':
+                raise ValueError(f'database {ordinary_database.name} engine must be Ordinary')
 
-        if atomic_name is None:
-            return
+            atomic_database.create(engine = 'Atomic')
+            tables = ordinary_database.tables
 
-        for table in tables:
-            ordinary_table = f'{ordinary_name}.{table}'
-            atomic_table = f'{atomic_name}.{table}'
-            self.__rename_table(ordinary_table, atomic_table)
+            for table in tables:
```

Comment:
1) You can call `move_tables` here
2) Try to unite logic in this function with logic in `__finished_previous_sessions`. Ideally, `__finished_previous_session` should just return bool if it needs to revert, continue or cancel